pkg/varlinkapi/containers_create.go

```diff
@@ -25,7 +25,7 @@ func (i *LibpodAPI) CreateContainer(call iopodman.VarlinkCall, config iopodman.C
 	rtc := i.Runtime.GetConfig()
 	ctx := getContext()
 
-	newImage, err := i.Runtime.ImageRuntime().New(ctx, config.Image, rtc.SignaturePolicyPath, "", os.Stderr, nil, image.SigningOptions{}, false, nil)
+	newImage, err := i.Runtime.ImageRuntime().New(ctx, config.Image, rtc.SignaturePolicyPath, "", os.Stderr, nil, image.SigningOptions{}, false, nil, false)
```

Comment:
Can we expose the new bool over the varlink API?

Reply:
on it

Reply:
this isnt sufficient.  you must update the .varlink file (cmd/podman/varlink/) for the bool.  and run varlink_api_generate to get the docs updated.  hit me up on irc with questions.